sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -72,11 +74,16 @@ class RocksDB(
     loggingId: String = "",
     useColumnFamilies: Boolean = false) extends Logging {
 
+  private val colFamilyNameToIdMap = new ConcurrentHashMap[String, Short]()
```

Comment:
Hmm - so we have this map in 2 places now ? in the store as well as the provider ? could we remove one of them ? maybe the one in the provider ?